src/utils/index.js

```diff
@@ -78,6 +78,33 @@ export const replaceMatches = (str, toReplace) => {
   }
 };
 
+/**
+ * Replace any parts of a string that match the '<' & '>' except '<b>' & '</b>'
+ * @param {String} str The string to check
+ * @returns {String} The string with or without replaced values
+ */
+export const replaceAngleBracketMatches = (str) => {
+  if (str?.length > 0) {
+    // Handling string like '<lambda>' or '<partial>' in 3 steps
+    // 1. replacing all '<b>' & '</b>' with unique '@$1$@' & '@$2$@' respectively
+    // 2. replacing all '<' & '>' with '&lt;' & '&gt;' respectively
+    // 3. replacing back all '@$1$@' & '@$2$@' with <b> & </b> respectively
+    const stgWithoutBTag = str
```

Comment:
Tiny nit: should `stgWithoutBTag` be `strWithoutBTag` instead? You use `str` elsewhere in the function.

Reply:
Done.